corehq/apps/export/dbaccessors.py

```diff
@@ -0,0 +1,32 @@
+
+
+def get_latest_case_export_schema_id(domain, case_type):
+    from .models import ExportDataSchema
+
+    key = [domain, 'CaseExportDataSchema', case_type]
+    result = ExportDataSchema.get_db().view(
+        'schemas_by_xmlns_or_case_type/view',
+        startkey=key + [{}],
+        endkey=key,
+        include_docs=False,
+        limit=1,
+        reduce=False,
+        descending=True,
```

Comment:
so it turns out it's easier to have them not be unique. when merging the schemas, the merge returns a new instance of the object (just easier to manage multiple merging schemas. and i like the idea of not mutating the object). i don't imagine this operation will happen too often, only on creation of a new export. Thus in order to get the most recent one, we need to use descending == True